proj-sys/build.rs

```diff
@@ -72,6 +72,7 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
 // returns the path of "include" for the built proj
 fn build_from_source() -> Result<std::path::PathBuf, Box<dyn std::error::Error>> {
     eprintln!("building libproj from source");
+    println!("cargo:rustc-cfg=bundled_build");
```

Comment:
What is this used for?

Reply:
It conditionally enables this `extern crate` statement here: https://github.com/georust/proj/pull/190/files#diff-bdc7b35321c79a3531cffc03776c83af67e9481bf67db7f7a94540dd559b0036R32, which in turn conditionally controls whether or not we want to link (statically or dynamically) libsqlite3 to the final binary.

Reply:
My memory is short and my ctrl+f is weak. Thank you.